UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Models/SalesCatalogue/Products.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace UKHO.ExchangeSetService.Common.Models.SalesCatalogue
+{
+    public class Products
+    {
+        public string ProductName { get; set; }
+        public int? EditionNumber { get; set; }
+        public List<int?> UpdateNumbers { get; set; }
+        public Cancellation Cancellation { get; set; }
+        public int? FileSize { get; set; }
+    }
+
+    public class Cancellation
+    {
+        public int? EditionNumber { get; set; }
+        public int? UpdateNumber { get; set; }
+    }
+}
```

Comment:
If somebody was looking for the Cancellation class it would be easier to find if it was on its own file. Are these two classes grouped together for a reason?

Reply:
Did respective changes. Moved all sub class to individual class

Reply:
Thanks